raft_test.go

```diff
@@ -2316,3 +2317,24 @@ func TestRaft_ReloadConfigValidates(t *testing.T) {
 //
 // Storage errors handled properly.
 // Commit index updated properly.
+
+func TestRaft_InstallSnapshot(t *testing.T) {
```

Comment:
super nit: this test is specifically only testing the behaviour for invalidly formed peers in a request. Is is worth giving it a more specific name?